admin_manual/configuration/server/logging_configuration.rst

```diff
@@ -9,13 +9,13 @@ Parameters
 
 Logging levels range from **DEBUG**, which logs all activity, to **FATAL**, which logs only fatal errors.
 
-* **0**: DEBUG: All activity; the most detailed logging.
-* **1**: INFO:  Activity such as user logins and file activities, plus warnings, errors, and fatal errors.
-* **2**: WARN:  Operations succeed, but with warnings of potential problems, plus errors and fatal errors.
-* **3**: ERROR: An operation fails, but other services and operations continue, plus fatal errors.
-* **4**: FATAL: The server stops.
+* **0**: DEBUG: Debug, informational, warning, and errors messages, and fatal issues.
+* **1**: INFO:  Informational, warning, and errors messages, and fatal issues.
+* **2**: WARN:  Warning, and errors messages, and fatal issues.
+* **3**: ERROR: Errors messages and fatal issues.
```

Comment:
In 0/1/2/3 change "errors messages" to "error messages"

Reply:
Thanks.